File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/pattern7.java
import java.util.Scanner;

public class pattern7 {

    public static void main(String[] args) {
        try (Scanner read = new Scanner(System.in)) {
            int n = read.nextInt();
            printDiamond(n);
        }
    }

    public static void printDiamond(int n) {
        // Your code here
        for (int i = 1; i <= n; i++) {
            for (int j = n; j > i; j--) {
                System.out.print(" ");
            }
            for (int k = 1; k <= i; k++) {
                System.out.print("* ");
            }
            System.out.println();
        }
        for (int i = n; i >= 1; i--) {
            for (int j = i; j < n; j++) {
                System.out.print(" ");
            }
            for (int k = i; k >= 1; k--) {
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}

File: SecondYear/ksh1710_KshitizAgarwal_2226cse1033_2/Week_1/pattern7.cpp
void printDiamond(int n) {
	     for (int i = 1; i <= n; i++){
	         
             for (int j =n-1; j >= i; j--){
                  cout<<" ";
               }
               for(int k=1;k<=i;k++){
                   cout<<"* ";
               }
            cout << endl;
         }
         
        for (int i = 1; i <= n; i++){
            
             for (int j =2; j <= i; j++){
                  cout<<" ";
                }
             for(int k=n;k>=i;k--){
                   cout<<"* ";
                }
            cout << endl;
         }
    }